File: Proyecto1-Servidor/src/main/java/com/dmaldonado/cliente/socket/message/Message.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.dmaldonado.cliente.socket.message;

import java.io.Serializable;

/**
 *
 * @author ACER
 */
public class Message implements Serializable{
    private String message;

    private static final long serialVersionUID = 6529685098267757690L;

    public Message() {
    }

    public Message(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "SimpleMessage{" +
                "message='" + message + '\'' +
                '}';
    }
}

File: Proyecto1Cliente/app/src/main/java/com/dmaldonado/cliente/socket/message/Message.kt
package com.dmaldonado.cliente.socket.message

class Message (val message: String) : java.io.Serializable  {
    companion object {
        private const val serialVersionUID = 6529685098267757690L
    }

    override fun toString(): String {
        return "SimpleMessage(message='$message')"
    }
}